manifests/module_plugin.pp

```diff
@@ -71,24 +71,34 @@
     }
   }
 
+  $_owner = $owner ? {
```

Comment:
Oh, I might've been thrown off when I was trying other ways to do undef. I'll give it a try.